README.md

```diff
@@ -93,6 +119,20 @@ Configurations:
 
 - `verbose`: print more detailed messages if true.
 
+- `check`: Whether to scan for compatibility issues (to improve execution speed). The default is true.
```

Comment:
Should be `process`

Reply:
I don't think so. This argument controls if we want to "check" compatibility issues